lib/NegativeAcksTracker.cc

```diff
@@ -88,9 +88,9 @@ void NegativeAcksTracker::handleTimer(const ASIO_ERROR &ec) {
             break;
         }
 
-        auto ledgerMap = it->second;
+        auto &ledgerMap = it->second;
         for (auto ledgerIt = ledgerMap.begin(); ledgerIt != ledgerMap.end(); ++ledgerIt) {
-            auto entrySet = ledgerIt->second;
+            auto &entrySet = ledgerIt->second;
```

Comment:
To avoid value copy, it's better to write `const auto& x = expression` whenever possible, i.e. except for the cases that:
- You want to get a mutable reference to modify the value that `expression` refers.
- You want to perform the value copy explicitly, which usually happens in the case that an immutable is passed to a different thread (typically use cases of `shared_ptr`)